### ISSUE TITLE
[NEW ALGORITHM]  Adding Bitonic Sort Algorithm

### ISSUE THREAD

**Issue by PrAyAg9** [AlgoGenesis/C#627]
## Issue will be closed if:

> 1) You mention more than one algorithm. You can create a separate issue for each algorithm once the current one is completed.  
> 2) You propose an algorithm that is already present or has been mentioned in a previous issue.  
> 3) You create a new issue without completing your previous issue.

**Note:** These actions will be taken seriously. Failure to follow the guidelines may result in the immediate closure of your issue.

---

### Name:  Bitonic Sort
```[NEW ALGORITHM]```  

### About:  
Propose a new algorithm to be added to the repository

Bitonic Sort is a parallel sorting algorithm that operates on sequences that are bitonic, meaning they are monotonically increasing and then monotonically decreasing. The algorithm first creates a bitonic sequence from the input array and then recursively sorts the array by merging bitonic sequences. This method is particularly effective in parallel processing environments.

---

### Labels:  
```new algorithm, gssoc-ext, hacktoberfest, level1```

---

### Assignees:  
- [ ✅] Contributor in GSSoC-ext
- [ ✅] Want to work on it
